Apsitvarkom.Api/Controllers/PollutedLocationController.cs

```diff
@@ -0,0 +1,95 @@
+﻿using Apsitvarkom.Models;
+using Apsitvarkom.Models.DTO;
+using Microsoft.AspNetCore.Mvc;
+namespace Apsitvarkom.Api.Controllers;
+
+[ApiController]
+[Route("/api/[controller]")]
+public class PollutedLocationController : ControllerBase
+{
+    private readonly ILogger<PollutedLocationController> _logger;
+
+    public PollutedLocationController(ILogger<PollutedLocationController> logger)
+    {
+        _logger = logger;
+    }
+
+
+    private static IEnumerable<PollutedLocationDTO> pollutedLocations = new List<PollutedLocationDTO>
+        {
+            new PollutedLocationDTO
+            {
+                Id = "5be2354e-2500-4289-bbe2-66210592e17f",
+                Location = new LocationDTO
+                {
+                    Latitude = 12.1651,
+                    Longitude = 121.151,
+                },
+                Radius = 10,
+                Severity = "MODERATE",
+                Spotted  =  DateTime.Now.ToString("yyyy-MM-dd HH:mm:ss.ttt"),
+                Progress = 15,
+                Notes = "This is test notes"
+
+            }
+        };
+
+    [HttpGet("{id}")]
+    public ActionResult<PollutedLocationDTO> GetById(string id)
+    {
+        var pollutedLocation = pollutedLocations.FirstOrDefault(PollutedPlace => PollutedPlace.Id == id);
+        if (pollutedLocation == null)
+        {
+            return BadRequest("Polluted location is not found - wrong id provided or id is not provided at all");
+        }
+        return Ok(pollutedLocation);
+    }
+
+    [HttpGet]
+    [Route("/[controller]/All")]
```

Comment:
on line 37 you use HttpGet("{id}"), but here you use HttpGet and then Route. Why not have it more consistently?

Reply:
As you have asked for clarification on this one, in some places you use `[HttpGet("{id}")]` and in this case you use `[HttpGet]` together with `[Route("/[controller]/All")]`. Why use two different routing approaches, instead of using `[HttpGet("All")` (at least I believe that should work).